src/Conversion/KrnlToAffine/ConvertKrnlToAffine.cpp

```diff
@@ -535,11 +535,21 @@ static LogicalResult interpretOperation(Operation *op, OpBuilder &builder,
     SmallVector<AffineForOp, 1> loopsToTile = {
         llvm::cast<AffineForOp>(loopRefToOp[blockOp.getLoop()])};
 
-    if (failed(tilePerfectlyNested(
-            loopsToTile, blockOp.getTileSizeAttr().getInt(), &tiledLoops))) {
+    int64_t step = blockOp.getTileSizeAttr().getInt();
+    if (failed(tilePerfectlyNested(loopsToTile, step, &tiledLoops))) {
       return failure();
     }
 
+    if (blockOp.getResult(1).use_empty()) {
```

Comment:
Just for my learning. Is this happened because of manually unrolling when generating krnl iterations?

Reply:
Got it. Thanks!